code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -138,10 +138,10 @@
 
 /obj/item/reagent_containers/hypospray/medipen/examine()
 	. = ..()
-	if(reagents?.reagent_list.len)
-		. += span_notice("It is currently loaded.")
+	if(length(reagents?.reagent_list))
+		. += span_notice("There\'s a small LCD screen on the side of the medipen which reads, 'WARNING: This medipen contains [pretty_string_from_reagent_list(reagents.reagent_list, TRUE, ", ", TRUE, TRUE)]. Do not use if allergic to any listed chemicals.' in small text.") //cheapskates at nanotrasen couldnt ball out for LED
```

Comment:
oh wait actually forgot. so since this is a child it will inherit the examine message from the parent. we'll need to make a new proc to handle this. call it /obj/item/reagent_containers/hypospray/medipen/proc/examine_text_lcd_output or something. put the output of that in this spot, and then change what each subtype says in that.

Reply:
Tested it and I dont think youre right if you are saying what I think youre saying, the two medipens which I gave a custom examine text have their custom examine texts still and hyposprays dont recieve this examine text

Reply:
All the children inheriting the examine message from the parent is the intended output I ensure, there's two medipens which I don't want to have this effect and it's easy to override them with 2 lines of code.